fitbenchmarking/fitbenchmarking
# Normalise naming conventions for minimizers

#### Description of Work
Use fitting result to generate minimizer names
Only pass jac and hess as strings to fitting result (rather than the objects)
Ensure any strings which are required to be unique use both software and minimizer (and jac and hess)
Remove unused return values from fitting_benchmark.py
Indent hessians into loop so running 2 hessian methods will use both hessian methods
Move alogrithm type from options into result

Fixes #1009


#### Testing Instructions

1. Make some varied results - multiple jacs, multiple hessians, etc.
   - [x] Output stream during run looks correct
   - [x] Table headers look correct
   - [x] Fitting reports appear correct (url, text, plots)
   - [x] Summary pages appear correct
2. Try running with mantid - BFGS and scipy - BFGS
   - [ ] Output stream during run looks correct
   - [ ] Table headers look correct
   - [ ] Fitting reports appear correct (url, text, plots)
   - [ ] Summary pages appear correct

Function: Does the change do what it's supposed to?
Tests: Does it pass? Is there adequate coverage for new code?
Style: Is the coding style consistent? Is anything overly confusing?
Documentation: Is there a suitable change to documentation for this change?
